### PR TITLE
Feature helpstate

### PR DESCRIPTION
**Pull request by ric-jon**
New State for the game, HelpState, which doubles as the pause screen. Instructions and Information is displayed here. <feature

### CODE REVIEW

File: src/main/java/nl/github/martijn9612/fishy/Main.java [hunk 1]
```diff
@@ -1,14 +1,11 @@
 package nl.github.martijn9612.fishy;
 
+import nl.github.martijn9612.fishy.states.*;
```
Comment (MilleIT) [inline src/main/java/nl/github/martijn9612/fishy/Main.java]: you shouldn't use stars when importing, just import individually.

File: src/main/java/nl/github/martijn9612/fishy/states/LevelState.java [hunk 2]
```diff
@@ -1,11 +1,6 @@
 package nl.github.martijn9612.fishy.states;
 
-import org.newdawn.slick.Color;
-import org.newdawn.slick.GameContainer;
-import org.newdawn.slick.Graphics;
-import org.newdawn.slick.Image;
-import org.newdawn.slick.SlickException;
-import org.newdawn.slick.Sound;
+import org.newdawn.slick.*;
```
Comment (MilleIT) [inline src/main/java/nl/github/martijn9612/fishy/states/LevelState.java]: here also, no stars

**Comment by MilleIT** [comment]
except for my other checkstyle comments the code is good, maybe split some methods up, try to keep the LOC per method under 30

**Comment by lenigoor** [comment]
Interesting new feature for our game! You're paying attention to code style and documenting methods, which is nice. However, also make sure your methods aren't too lenghty, for example the `render()` method in the `HelpState` class should be split into more methods.

**Comment by lenigoor** [comment]
Nice, way to go. The branch still has merge conflicts that should be resolved, before we can merge it.

**Comment by lenigoor** [comment]
That are a lot of changes for resolving a merge conflict! However, since I recognise the code that I wrote in a previous branch, I can see the correct changes were cherry-picked from development. Will merge the pull-request.

**Pull request merged**